test/index.test.js

```diff
@@ -80,6 +82,28 @@ describe('Sentry', function() {
         analytics.assert.deepEqual(window.RavenConfig.config, config);
       });
 
+      it('should allow and set custom versions', function() {
+        var config = {
+          logger: options.logger,
+          serverName: options.serverName,
+          whitelistUrls: options.whitelistUrls,
+          ignoreErrors: options.ignoreErrors,
+          ignoreUrls: options.ignoreUrls,
+          includePaths: options.includePaths,
+          maxMessageLength: options.maxMessageLength,
+          release: '2.4.0'
```

Comment:
Ah, that's already tested in the other settings (this is much more of a contract test that we compare to - the main settings are separate). But I totally agree with changing sentry directly and keeping it sane, pushing a test fix!